.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1 +1 @@
-distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.5.4/apache-maven-3.5.4-bin.zip
\ No newline at end of file
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.3.9/apache-maven-3.3.9-bin.zip
```

Comment:
please revert .. if anything we want to upgrade to a newer wrapper version.. not go back older..

Reply:
It's been reverted to 3.5.4. I pushed it back to 3.3.9 because many IDEs are still baselined on Maven 3.3, but I know IntelliJ at least will be baselining on Maven 3.6.0 in its next release.